apps/modeler/include/pcl/apps/modeler/main_window.h

```diff
@@ -128,8 +128,7 @@ namespace pcl
         friend class AbstractItem;
 
         MainWindow();
-        MainWindow(const MainWindow &) {} // copy ctor hidden
-        MainWindow& operator=(const MainWindow &) { return (*this); } // assign op. hidden
+        MainWindow& operator=(const MainWindow &) = delete;
```

Comment:
Correct and wrong at same time:
- Wrong: You change behavior.
- Right: Copying Ui classes is indeed a bad idea, except you are doing a deep copy of pointer.

As far as I rember clang-tidy introduced `= default` and not `=default`, or? So we should use `=delete` (without whitespace).

Reply:
Well the project compiles, meaning that `operator=` was not used anywhere. According to the comment the original intention was to disable it, we do it explicitly now.

> So we should use =delete (without whitespace).

Don't understand this. Can you expand?

Reply:
> Don't understand this. Can you expand?

Sorry my fault. Mixed our project coding style with coding style of PCL clang-tidy. Had expected [this](https://clang.llvm.org/extra/clang-tidy/checks/modernize-use-equals-delete.html) added `=delete` and not `= delete` (the whitespace between `=` and `delete` is the difference).